cmd/kubelet/app/server.go

```diff
@@ -197,6 +197,7 @@ is checked every 20 seconds (also configurable with a flag).`,
 
 			if cleanFlagSet.Changed("pod-infra-container-image") {
 				klog.InfoS("--pod-infra-container-image will not be pruned by the image garbage collector in kubelet and should also be set in the remote runtime")
+				cmd.Flags().MarkDeprecated("pod-infra-container-image", "--pod-infra-container-image will be removed in 1.28. Image garbage collector will get sandbox image information from CRI.")
```

Comment:
nit: this is no longer accurate. The field will likely be removed in 1.30 at this point